core/packages/PackageInfo.cc

```diff
@@ -344,7 +345,8 @@ optional<core::AutocorrectSuggestion> PackageInfo::addExport(const core::GlobalS
 
     core::AutocorrectSuggestion suggestion(
         fmt::format("Export `{}` in package `{}`", newExportName, mangledName_.owner.show(gs)),
-        {{insertionLoc, fmt::format("\n  {}", exportLine)}});
+        {{insertionLoc, fmt::format("\n  {}", exportLine)}}, false /* isDidYouMean */,
+        false /* shouldSkipWhenAggregated */);
```

Comment:
Intentionally set as `false`. Will change to `true` when we have a `aggregateMissingExportForFile`.